datalad/core/distributed/push.py

```diff
@@ -229,7 +229,13 @@ def __call__(
                     if sr
                     else 'No targets configured in dataset.'))
             return
-
+        potential_remote = False
+        if not to and path:
```

Comment:
May be that should be relevant only, if the path doesn't actually exist?

Reply:
good point. I moved it down into the ``if not matched_anything`` section.